src/parcsr_ls/par_mgr_setup.c

```diff
@@ -1197,9 +1197,9 @@ hypre_MGRSetup( void               *mgr_vdata,
                   }
                   else
                   {
-                     if (my_id == 0)
+                     if (my_id == 0 && print_level > 1)
                      {
-                        hypre_printf("Error!!! Invalid AMG setup for user-prescribed F-relaxation.\n");
+                        hypre_error_w_msg(HYPRE_ERROR_GENERIC, "Error!!! Invalid AMG setup for user-prescribed F-relaxation.\n");
                         hypre_error(1);
```

Comment:
You can delete this line BTW since you've already signaled the error above.

Reply:
@rfalgout just confirming you mean the hypre_error(1); line?

Reply:
Yes.  Sorry for not being clear.

Reply:
Got it. Thanks!